kroxylicious-filters/kroxylicious-multitenant/src/main/java/io/kroxylicious/proxy/filter/multitenant/MultiTenant.java

```diff
@@ -22,12 +25,12 @@ public class MultiTenant implements FilterFactory<MultiTenantConfig, MultiTenant
     private static final MultiTenantConfig DEFAULT_TENANT_CONFIG = new MultiTenantConfig(null);
 
     @Override
-    public MultiTenantConfig initialize(FilterFactoryContext context, MultiTenantConfig config) {
+    public @Nullable MultiTenantConfig initialize(FilterFactoryContext context, @Nullable MultiTenantConfig config) {
         return config;
     }
 
     @Override
-    public MultiTenantFilter createFilter(FilterFactoryContext context, MultiTenantConfig configuration) {
+    public Filter createFilter(FilterFactoryContext context, @Nullable MultiTenantConfig configuration) {
```

Comment:
The change in the return type here is because `MultiTenantFilter` is not `public`, but we were using it on a public method => warning. I thought it better to just claim to return a `Filter` than making the filter `public`.